security/fma-isthmus-l2-pectra.md

```diff
@@ -0,0 +1,207 @@
+>   - **Final 👍:** Security will transition the status of the document to Final once all action items are completed.
+
+> [!TIP]
+> Guidelines for writing a good analysis, and what the reviewer will look for:
+>
+> - Show your work: Include steps and tools for each conclusion.
+> - Completeness of risks considered.
+> - Include both implementation and operational failure modes
+> - Provide references to support the reviewer.
+> - The size of the document will likely be proportional to the project's complexity.
+> - The ultimate goal of this document is to identify action items to improve the security of the project. The FMA review process can be accelerated by proactively identifying action items during the writing process. -->
+
+## Introduction
+
+This document covers implementation of Pectra features in the OP stack. This requires a hard fork due to new transaction types like EIP-7702 transactions. This does not cover generic hardfork related security concerns, which will be addressed in a separate document.
+
+Below is a list of EIPs that are enabled with Pectra, and a quick description of spec changes needed to support the EIP.
+
+- EIP-7702: Set code transaction
+  - [Added authorization list to span batch format](https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/derivation.md#span-batch-updates)
+- EIP-2537: BLS12-381 precompiles
+  - [Added input limits and specify acceleration for new precompiles](https://github.com/ethereum-optimism/specs/blob/a795bb1f8c34410c47f05d4c3614482361db64b1/specs/protocol/isthmus/exec-engine.md#evm-changes)
+- EIP-2935: Block hashes contract predeploy
+  - [Added network upgrade transaction](https://github.com/ethereum-optimism/specs/blob/a795bb1f8c34410c47f05d4c3614482361db64b1/specs/protocol/isthmus/derivation.md#eip-2935-contract-deployment)
+- EIP-7251: Increase the MAX_EFFECTIVE_BALANCE
+  - Only affects execution clients
+- EIP-7623: Increase calldata cost
+  - Only affects execution clients
+- EIP-7002: Execution layer triggerable withdrawals
+  - Not applicable to OP
+  - Requests sent to this contract are ignored, and the contract is not deployed by default
+- EIP-6110: Supply validator deposits on chain
+  - Not applicable to OP
+  - Requests sent to this contract are ignored, and the contract is not deployed by default
+- EIP-7685: General purpose execution layer requests
+  - No requests are currently applicable to OP, so the [execution requests array is always empty in Isthmus](https://github.com/ethereum-optimism/specs/blob/b5e0fa98881171f658f782597a46b641e8f3dfd0/specs/protocol/exec-engine.md#engine_newpayloadv4).
+
+## Failure Modes and Recovery Paths
+
+### FM1: Chain halt due to upgrade transaction failing
+
+- **Description:** The chain could halt if a network upgrade transaction can't be included in the upgrade block.
```

Comment:
Which transaction upgrade are speaking here? Can we details more in the description?
How does look like an "upgrade transaction"? Do we have a doc or referrence to point to this?